sbnobj/Common/Calibration/TrackCaloSkimmerObj.h

```diff
@@ -98,6 +101,42 @@ namespace sbn {
      iproc(-1) {}
   };
 
+  struct TrueHit {
+    int16_t cryo; //!< Cryostat of hit
+    int16_t tpc; //!< TPC of hit
+    int16_t plane; //!< Plane of hit
+    int16_t wire; //!< Wire of hit
+    int16_t channel; //!< Channel of hit
```

Comment:
`channel` needs to accommodate more than `32767` channels:
```suggestion
    uint16_t channel; //!< Channel of hit
```
or set it to `uint32_t` (or simply `unsigned int`, if `raw::ChannelID` is not desired).
(note that such change may trigger another version in `classes_defs.xml`, which would then need to be trimmed again of the intermediate version)

Reply:
I would like to be able to set the channel to -1 to allow for a "bad" value. I can revert the type to an int32 in that case.

Reply:
Yes, or simply `int`.